cogs/greetings.py

```diff
@@ -41,7 +42,7 @@ async def on_raw_reaction_add(self, payload):
 
     @commands.Cog.listener()
     async def on_raw_reaction_remove(self, payload):
-        if payload.channel_id == constantvariables.welcome_channel_id:
+        if payload.channel_id == constantvariables.welcome_channel_id and payload.message_id == self.welcome_message_id:
             if "👍" in str(payload.emoji):
```

Comment:
👍 seems to be a "hardcoded" emoji that can be changed/customized per server. Let's make this a constant as well.